src/sdk-reference/js/6/user/getProfiles/snippets/getProfiles.js

```diff
@@ -0,0 +1,26 @@
+const credentials = { username: 'foo', password: 'bar' };
+
+try {
+  await kuzzle.auth.login('local', credentials);
+
+  const user = await kuzzle.auth.getCurrentUser();
+
+  const profiles = await user.getProfiles();
+  console.log(profiles);
+  /*
+    [
+      Profile {
+        _id: 'default',
+        policies: [
+          {
+            roleId: 'default'
+          }
+        ]
+      }
+    ]
+  */
+
+  console.log(`Successfully retrieved ${profiles.length} profiles.`);
```

Comment:
This console.log should be removed, and the previous one should be tested instead: it's clearer that way (less code to read for users), and this makes the test more fiable.

```suggestion
```